Library/test-helpers/lib.sh

```diff
@@ -1376,6 +1376,10 @@ limeCreateTestPolicy() {
 
     # create policy.json...
     rlRun "limeCopyKeylimeFile --install cmd/convert_runtime_policy.py && python3 convert_runtime_policy.py -a allowlist.txt -e excludelist.txt -o policy.json"
+    # create signed policies and keys
+    rlRun "limeCopyKeylimeFile --install cmd/sign_runtime_policy.py && python3 sign_runtime_policy.py -r policy.json -p dsse-ecdsa-privkey.key -b ecdsa -o policy-dsse-ecdsa.json"
```

Comment:
@mbestavros Where is the dsse-ecdsa-privkey.key coming from?

Reply:
@kkaarreell It's generated as part of the signing script. A user can pass an ECDSA private key, or the tool can create one if not specified.

Reply:
In fact I can see that in tests we copy scripts here so maybe you can simply run it from this path
https://github.com/RedHat-SP-Security/keylime-tests/blob/main/setup/install_upstream_keylime/test.sh#L91